<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointerCalibView</class>
 <widget class="QWidget" name="PointerCalibView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="ctkCoordinatesWidget" name="m_TipOriginSpinBoxes" native="true"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="m_SensorToWorldLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>sensor to world</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="m_MapsToLabel">
         <property name="text">
          <string>maps to</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="m_TipOriginLabel">
         <property name="text">
          <string>tip origin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QmitkDataStorageComboBox" name="m_SensorToWorldNode"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_PointerToWorldLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>pointer to world</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QmitkDataStorageComboBox" name="m_PointerToWorldNode"/>
       </item>
       <item row="3" column="1">
        <widget class="ctkCoordinatesWidget" name="m_MapsToSpinBoxes" native="true"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="m_ScalingLabel">
         <property name="text">
          <string>scaling</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="m_ScalingParametersLabel">
         <property name="text">
          <string>scaling in mm</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="m_SaveToFileButton">
         <property name="text">
          <string>Save to file</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="m_RigidLabel">
         <property name="text">
          <string>rigid</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="ctkMatrixWidget" name="m_RigidMatrix" native="true"/>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="m_FiducialRegistrationErrorLabel">
         <property name="text">
          <string>fiducial registration error</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="m_PointsTextBox"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCoordinatesWidget</class>
   <extends>QWidget</extends>
   <header location="global">ctkCoordinatesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkMatrixWidget</class>
   <extends>QWidget</extends>
   <header>ctkMatrixWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
